src/configuration/js-team/linux64.js

```diff
@@ -0,0 +1,334 @@
+import SETTINGS from '../perfherderSettings';
+
+const { frameworkIds } = SETTINGS;
+
+const Linux64 = {
+  label: 'Linux 64bit',
+  benchmarks: {
+    kraken: {
+      compare: {
+        kraken: {
```

Comment:
Let's make sure that on **every** benchmark we list in `compare` we have the `platform` and `'project` properties defined. Like you do right below.